src/NodaTime/Instant.cs

```diff
@@ -526,6 +533,13 @@ public static Instant Max(Instant x, Instant y)
             BclEpoch.PlusTicks(dateTimeOffset.Ticks - dateTimeOffset.Offset.Ticks);
 
         /// <summary>
+        /// Converts a Julian Day Number into a new Instant representing the same instant in time
+        /// </summary>
+        /// <param name="JDN">a <see cref="double"/> containing the number of whole and fractional days since the Julian Epoch (Noon January 1st, 4713 BCE)</param>
+        /// <returns>An <see cref="Instant"/> value representing the same instant in time as the given <see cref="double"/>.</returns>
+        public static Instant FromJulianDayNumber(double JDN) => UnixEpoch.PlusTicks(Convert.ToInt64((JDN - 2440587.5) * 86400) * TicksPerSecond);
```

Comment:
I don't think I'd use the Unix epoch here at all. I'd create a property of JulianEpoch representing noon on January 1st 4713 BCE (in UTC) and use that for both calculations.